apps/api/src/app/subscribers-v2/subscribers.controller.ts

```diff
@@ -11,7 +11,7 @@ import {
   UseInterceptors,
 } from '@nestjs/common';
 import { ApiOperation, ApiTags } from '@nestjs/swagger';
-import { ExternalApiAccessible, UserSession } from '@novu/application-generic';
+import { CreateOrUpdateSubscriberUseCase, ExternalApiAccessible, UserSession } from '@novu/application-generic';
```

Comment:
this old usecase doesnt support the `timezone` field, you'll need to update that usecase to have the timezone add/edit functionality